packages/component-model-survey-response/entities/surveyResponse/index.test.js

```diff
@@ -0,0 +1,51 @@
+const { createTables } = require('@elifesciences/component-model')
+const uuid = require('uuid')
+const SurveyResponse = require('.')
+const replaySetup = require('../../../../test/helpers/replay-setup')
+
+replaySetup('success')
+
+const createSurveyResponse = params =>
+  new SurveyResponse({
+    response: params || {
+      questions: [
+        {
+          id: '1',
+          question: 'Question 1',
+        },
+      ],
+      answers: [
+        {
+          questionId: '1',
+          answer: 'Answer 1',
+        },
+      ],
+    },
+    manuscriptId: uuid(),
+    surveyId: uuid(),
+  }).save()
+
+describe('AuditLog', () => {
+  beforeEach(async () => {
+    await createTables(true)
+  })
+
+  it('should save to the database', async () => {
+    const surveyResponse = await createSurveyResponse()
+    expect(surveyResponse.id).toBeTruthy()
```

Comment:
Can we check this is a UUID?

Reply:
Wooo regex time!

Reply:
Seeing as this is being set by the `BaseModel` I would think that should be tested in the `component-model` unit tests?